### PR TITLE
Create cyhy logging module

### PR DESCRIPTION
**Pull request by felddy**
#  #

## 🗣 Description ##

This PR introduces initial logging configuration for the `cyhy-logging` repository, including setting up a basic logging infrastructure that aligns with our current standards.

This configuration aims to streamline log collection and standardize logging across different components of CyHy.

## 💭 Motivation and context ##

The CyHy system requires consistent and centralized logging to effectively monitor activities, identify issues, and analyze the health of services.

This change establishes a consistent logging standard and integrates the logging service into the codebase, ensuring uniformity in logging practices across services. The approach was to implement a well-tested configuration using Python's `logging` module.

## 🧪 Testing ##

The changes were tested using unit tests to ensure that logging output follows the desired format and level settings. Additionally, the service was run in a test environment, and logs were inspected for proper content and format.

Tests were executed in a local Docker environment running Python 3.12. Logging was validated in both `stdout` and file outputs. Verified that different log levels were correctly captured.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] *All* future TODOs are captured in issues, which are referenced
      in code comments.
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
- [x] Tests have been added and/or modified to cover the changes in this PR.
- [x] All new and existing tests pass.

## ✅ Pre-merge checklist ##

- [x] Revert dependencies to default branches.
- [x] Finalize version.

## ✅ Post-merge checklist ##

- [ ] Create a release.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -58,15 +58,15 @@ def get_version(version_file):
     },
     # Author details
     author="Cybersecurity and Infrastructure Security Agency",
```
Comment (dav3r) [inline setup.py]: You missed this:
```suggestion
    author="Mark Feldhousen",
```

Comment (felddy) [reply setup.py]: Doh!  I clicked the commit suggestion button too furiously.  Now there is a craptacular commit message in there.  I'm going to fix it up.  Standby for rebase.

Comment (felddy) [reply setup.py]: Resolved: 
- https://github.com/cisagov/cyhy-logging/pull/1/commits/44c674985e89005a03e92ca6feb82b15a03eb804

**Review by dav3r — APPROVED** [review]
LGTM - strong work! 🦾 💼

**Review by jsf9k — APPROVED** [review]

**Pull request merged**